experiments/amg2023/experiment.py

```diff
@@ -12,9 +12,14 @@
 from benchpark.scaling import StrongScaling
 from benchpark.scaling import WeakScaling
 from benchpark.scaling import ThroughputScaling
+from benchpark.domaindecomposition import PerProcessDomains
 from benchpark.caliper import Caliper
 
-
+@PerProcessDomains(
+strong_scaling_policy="conserveglobalproblemsize",
+weak_scaling_policy="default",
+throughput_scaling_policy="default",
+)
```

Comment:
@pearce8 @scheibelp Implementation of per-process scaling policies `lib/benchpark/domaindecomposition.py::PerProcessDomains`